src/apps/admin/pages/MusicQueue.scss

```diff
@@ -0,0 +1,22 @@
+.music-queue_container h1 {
+  @include font-headline('sm', 600);
+}
+
+.music-queue_container {
+  display: flex;
+  flex-direction: column;
+  gap: 1%;
+}
+
+.next_up_container {
+  display: flex;
+  flex-direction: column;
+  gap: 2vh;
```

Comment:
I recommend doing gaps with `rem` instead of `vh`, there's some code in the base styles that resets `1rem` from being equal to `16px` to `10px`.  If you do gaps in `vh`, then if I were to minimize my screen and manually shrink the height of the window without changing the width, the gaps will get all weird (since it is based on height). Rems are based on font-size instead - which is a bit more stable.

Don't need to change it now, but you probably should switch it over eventually.

More on CSS units: <https://www.freecodecamp.org/news/absolute-and-relative-css-units/>